### ISSUE TITLE
Include pre-installed wordpress in docker

### ISSUE THREAD

**Issue by jeni1510** [docker-library/wordpress#751]
hello, I have a site made in wordpress and I would like to take it to a docker container to have several test work environments, is this possible with docker? or is it only possible to install new wordpress?

**Comment by wglambert** [comment]
Yep, see https://github.com/docker-library/docs/tree/master/wordpress#include-pre-installed-themes--plugins
https://github.com/docker-library/wordpress/discussions/623#discussioncomment-1006225
